provider/global_secondary_index.go

```diff
@@ -208,6 +215,11 @@ func dynamoDBGSICreate(d *schema.ResourceData, m interface{}) error {
 		},
 	}
 
+	if d.Get("billing_mode") == dynamodb.BillingModePayPerRequest {
+		// Clear provisioned throughput for PAY_PER_REQUEST billing mode.
```

Comment:
Should it error if `read_capacity` or `write_capacity` is set instead of just ignoring them?

Reply:
Done, and added argument validation to ensure consistency.

Reply:
this comment is not applicable anymore

Reply:
fixed, thanks!